app/assets/javascripts/school_administrator_dashboard/dashboard_components/school_administrator_dashboards.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {Switch, Route} from 'react-router-dom';
+
+import DashboardOverview from './dashboard_overview.jsx';
+import SchoolwideAbsences from './absences_dashboard/schoolwide_absences.jsx';
+import SchoolwideTardies from './tardies_dashboard/schoolwide_tardies.jsx';
+
+export default React.createClass({
+  displayName: 'SchoolAdministratorDashboards',
+
+  propTypes: {
+    serializedData: React.PropTypes.object.isRequired
+  },
+
+  render: function() {
```

Comment:
I'd suggest finishing off this PR and not solving this right now in this PR, but over time let's converge towards having routing all in one place.  This adds a second "layer" of routing beyond the main one in https://github.com/studentinsights/studentinsights/blob/master/ui/index.js#L27 and https://github.com/studentinsights/studentinsights/blob/master/ui/route.js.

Maybe let's keep moving and come back to "Convert JS routing to use React Router" in a separate issue later?

Reply:
@kevinrobinson I'm also open to using the existing router, but I got a little stuck. The problem I'm trying to solve is that I don't want to reload the data for the dashboards every time a user switches the view. If there's a straightforward way to do that without something like react-router, I'm happy to go that route. If you have some bright ideas and a few minutes for a call, I'd be glad to talk it through!

Reply:
@edavidsonsawyer Yeah I get you, and that seems great!  I think doing what you're doing here is a fine immediate-term solution, and no need to block and solve that differently now.

Another immediate-term solution could be having the component here hold a bit of state for which page should be shown, and then pass down a callback to the different dashboard components that let them change to another dashboard.  But I think what you're doing here is a good solution for now - my vote would be to just keep moving and ship it while calling out it's a thing to refactor later.